src/store/index.ts

```diff
@@ -0,0 +1,37 @@
+import Vuex from 'vuex';
+import Vue from 'vue';
+
+import app from './modules/app';
+import toolbar from './modules/toolbar';
+import { RootState } from '@/store/types';
+
+Vue.use(Vuex);
+
+/**
+ * The Vuex store
+ */
+const store = new Vuex.Store<RootState>({
+  modules: {
+    app,
+    toolbar,
+  },
+});
+
+if (module.hot) {
+  // accept actions and mutations as hot modules
```

Comment:
Hot module replacement is a feature the Vue provides. Basically when we make a code change, instead of having to reload the page, the changes we make are often capable of just propagating to the page. It will also maintain the previous state of the application, which can be really nice. Should only be enabled in development